addons/mail/static/src/new/core/composer_model.js

```diff
@@ -1,4 +1,4 @@
-/** @odoo-module */
+/** @odoo-module **/
```

Comment:
(new) guideline is single * (both at the start and at the end)
double ** is for jsdoc, and this is not jsdoc

Reply:
and at the end it should always be single *

Reply:
seriously, this is not important (also, the vast majority of the codebase uses `**`)

Reply:
This is definitely not a reason to block a PR, still I value harmony through our code. This point is worth making at least once, because now that I know about it, I will adopt the recommended style, resulting in more harmonious code without any extra effort.